win32ss/gdi/eng/device.c

```diff
@@ -361,6 +439,17 @@ EngpRegisterGraphicsDevice(
     EngReleaseSemaphore(ghsemGraphicsDeviceList);
     TRACE("Prepared %lu modes for %ls\n", pGraphicsDevice->cDevModes, pGraphicsDevice->pwszDescription);
 
+    /* HACK: already in graphic mode; display wallpaper on this new display */
```

Comment:
Is it necessary to duplicate what is being done in `co_IntInitializeDesktopGraphics()` ?

Reply:
Yes.
The hack in co_IntInitializeDesktopGraphics() displays wallpaper on all screens already available/installed when switching to graphic mode.
The hack here displays wallpaper on new screens installed after the switch to graphic mode.